toonz/sources/include/tgeometry.h

```diff
@@ -100,16 +130,22 @@ inline std::ostream &operator<<(std::ostream &out, const TPointT<T> &p) {
 
 typedef TPointT<int> TPoint, TPointI;
 typedef TPointT<double> TPointD;
+typedef TPoint4T<double> TPoint4D;
 
 #ifdef _WIN32
 template class DVAPI TPointT<int>;
 template class DVAPI TPointT<double>;
+template class DVAPI TPoint4T<double>;
 #endif
 
 template <class T>
 inline bool operator==(const TPointT<T> &p0, const TPointT<T> &p1) {
   return p0.x == p1.x && p0.y == p1.y;
 }
+template<class T>
+inline bool operator!=(const TPointT<T> &p0, const TPointT<T> &p1) {
+  return p0.x != p1.x && p0.y != p1.y;
```

Comment:
This change breaks Undo Move Cell, and potentially any place where  TPoint comparisons are done.

This should be:
>  return p0.x != p1.x **||** p0.y != p1.y;